app/api/email_utils.py

```diff
@@ -3,6 +3,7 @@
 from flask_mail import Message
 from flask import render_template
 
+import config
 from app.api.mail_extension import mail
 
 EMAIL_VERIFICATION_TOKEN_TIME_TO_EXPIRE = 86400  # 24 hours in seconds
```

Comment:
Umm! Sorry but last doubt, This says, email token expires in 24 hours, and we changed the mail description to verify within 48 hours?

What we want is they can verify there account within 48 hours,
But they can only use one link within 24 hours of sending it.

So, Maybe we need to edit that message in different way. Not exactly but something similar to this -

```
The activation link will be active for {{ EMAIL_VERIFICATION_TIME // 3600 }} hours.
You can verify your account within {{ threshhold // 3600 }} hours, otherwise your account will be automatically deleted.
```

Reply:
I understand, BTW I forgot to edit that comment.
So maybe let's just change the `THRESHOLD` to 24 hours?

Reply:
@isabelcosta @sara-02 what do you suggest?

Reply:
I'd like to add that it's impossible to pass values bigger than `23 hours`, `59 minutes` or `59 seconds` to `add_job()`. APScheduler library doesn't support it. That's why I convert it to days - that's the only time unit that doesn't crash it.

Reply:
That's a great thing you found! It seems a bit weird that you changed hours to days, but now that we know why. Totally cool! :sunglasses:

Reply:
@BadduCoder Changed threshold to 24 hours. 

Is there anything else I could improve?

Reply:
yes if you could please add this as a part of other schedulers instead of creating new one?